From: Yaroslav Yachmenov
Date: Sat, 16 Feb 2019 15:21:00 +0300
Subject: [PATCH] Show error message for 30 seconds and scroll it into view
<+>UTF-8
===================================================================
--- a/view/frontend/web/js/view/messages.js
+++ b/view/frontend/web/js/view/messages.js
@@ -67,8 +67,13 @@
             // Hide message block if needed
             if (isHidden) {
                 setTimeout(function () {
-                    $(self.selector).hide('blind', {}, 500);
-                }, 5000);
+                    $(self.selector).hide('blind', {}, 500);
+                }, 30000);
+                try {
+                    $(self.selector)[0].scrollIntoView();
+                } catch (error) {
+                    // Do nothing
+                }
             }
         }
     });
